xcp_d/utils/plot.py

```diff
@@ -1006,3 +1006,135 @@ def _get_tr(img):
     except AttributeError:  # Error out if not in cifti
         return img.header.get_zooms()[-1]
     raise RuntimeError("Could not extract TR - unknown data structure type")
+
+
+def plot_alff_reho_volumetric(output_path, filename, bold_file):
+    """Plot ALFF/ReHo mosaic plot for Niftis."""
+    from bids.layout import parse_file_entities
+    from nilearn import plotting as plott
+    from templateflow.api import get as get_template
+
+    space = str(parse_file_entities(bold_file)["space"])
+    try:
+        resolution = str(parse_file_entities(bold_file)["res"])
+    except Exception:
+        resolution = None
```

Comment:
```suggestion
    resolution = parse_file_entities(bold_file).get("res")
```

Since the parsed file entities should be a dictionary, you can just get `.get`, which will default to None if the key isn't in the dictionary.